openslides/utils/collection.py

```diff
@@ -1,25 +1,36 @@
 from django.apps import apps
+from django.core.cache import cache, caches
 
 
 class CollectionElement:
     @classmethod
     def from_instance(cls, instance, is_deleted=False):
         """
         Returns a collection element from a database instance.
+
+        This will also update the instance in the cache.
+
+        If is_deleted is set to True, the element is deleted from the cache.
         """
         return cls(instance=instance, is_deleted=is_deleted)
 
     @classmethod
-    def from_values(cls, collection_string, id, is_deleted=False):
+    def from_values(cls, collection_string, id, is_deleted=False, full_data=None):
```

Comment:
Ok. Please add a short note in the docstring, e. g. `The argument full_data may already contain all data. They are fetched out of the cache.`